internal/backend/mqttpubsub/backend.go

```diff
@@ -102,6 +111,14 @@ func NewBackend(c BackendConfig) (*Backend, error) {
 	opts.SetOnConnectHandler(b.onConnected)
 	opts.SetConnectionLostHandler(b.onConnectionLost)
 
+	//if b.config.LastWillPayload == "" {
+	//	mac := lorawan.EUI64
```

Comment:
Need some help here, how can I get the mac address ?

Reply:
Actually, there might be an issue. When the LoRa Gateway Bridge represents a single gateway, this works. However in a lot of setups, this is not the case. I also believe some gateway vendors have a custom packet-forwarder that simulates multiple gateways when dealing with multiple gateway boards (e.g. > 8 channels).

Maybe you were right that setting a custom payload would be a better idea, as then it is up to the end-user to decide "what to set".

Just to understand the use-case better, could describe the problem that this would solve? Note that most gateways send a periodic stats message (e.g. every 30 seconds) which could also be used as an indication if the gateway is up or down.

I do appreciate your effort (also to learn Go!). I just try to make sure that this will end up be a feature that has a general purpose. Adding features is easy, but as soon as people are going to use them, it becomes hard to change them.